### ISSUE TITLE
Fix code headers and run ID line script

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#509]
This is not a big deal but it seems like folks enjoy having their names on the individual files they've worked on via the code headers. The code header in the form of:

```python
# yellowbrick.target
# Implements visualizers related to the dependent (target) variable, y.
#
# Author:  Benjamin Bengfort <benjamin@bengfort.com>
# Created: Thu Jul 19 08:57:05 2018 -0400
# License: Apache-2.0
#
# ID: __init__.py [abcd123] benjamin@bengfort.com $
```

Is actually kind of a throwback from some of my earliest coding days when we had git commit hooks that did automatic checking for safety and linting (before the days of GitHub services that did this). The git commit hook would actually fill in all of the information except for the description line. 

I actually did dig up the script that did this and did some repairs to it. I'd like to make all the files in the package consistent with this header by running the script. Unfortunately that will make a lot of changes to a lot of files; and I don't want to do that in a PR - rather I think that we should do this on our next release as part of the version bump. 

CC: @rebeccabilbro @ndanielsen @lwgray

**Comment by bbengfort** [comment]
*Update*: In a recent call we decided to have the copyright line attributed to "The scikit-yb developers", the header should be updated to reflect this.

**Comment by bbengfort** [comment]
Fixed in #957

**Comment by rebeccabilbro** [comment]
Here's the [gist](https://gist.github.com/rebeccabilbro/4bde1df0bce8b0e241a90576e874ec21) for the ID line script
